virtcontainers/container.go

```diff
@@ -1359,8 +1359,8 @@ func (c *Container) updateCgroups(resources specs.LinuxResources) error {
 	}
 
 	// store new resources
-	c.state.Resources = r
-	if err := c.store.Store(store.State, c.state); err != nil {
+	c.config.Resources = r
```

Comment:
is valid to do this? can we modify the config.json ?

Reply:
Why not? The config.json is not the original OCI spec from other components(K8S or docker), it's another copy maintained by kata-runtime. I don't see a reason that we can't modify it.